index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="main-container">
+        <div class="header">
+            <div class="sub-header logo">
+                <a href="url"><img src="./logo-placeholder.png" alt=""></a>
+            </div>
+            <div class="sub-header menu">
+                <a href="url"><img src="./icon-menu.png" alt=""></a>
+            </div>
+        </div>
+        <div class="content-container">
+            <div class="title">
+                <h1>I'm Kazim<br>Glad to see you!</h1>
+            </div>
+            <div class="content">
+                <p>
+                    I'm a soffware developer! I can help you build a product, 
+                    feature or website Look through some of my work and experience!
+                    if you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.
+                </p>
+            </div>
+            <div class="social-media-title">
+                <h6>
+                    LET'S CONNECT
+                </h6>
+            </div>
+            <div class="social-media-icon">
+                <a href="url" id="twitter-icon"><img src="./icon-tiwtter.svg" alt=""></a>
+                <a href="url" id="linkedin-icon"><img src="./icon-linkedin.svg" alt=""></a>
+                <a href="url" id="m-icon"><img src="./icon-m.svg" alt=""></a>
+                <a href="url" id="github-icon"><img src="./icon-github.svg" alt=""></a>
+                <a href="url" id="free-icon"><img src="./icon-free.svg" alt=""></a>
+            </div>
+        </div>
+    </div>
```

Comment:
[REQUIRED] Kindly include a section tag to wrap the headline-section

[REQUIRED]Your background image is not pressed against the right edge just like the template

![image](https://user-images.githubusercontent.com/62938735/199930943-80bd3692-ea1e-460e-8dd7-f8cfa66c7af5.png)